scripts/kfctl.sh

```diff
@@ -157,9 +107,63 @@ createNamespace() {
   fi
 }
 
+# Generate all required components
+customizeKsApp() {
+  ks param set ambassador platform ${KUBEFLOW_PLATFORM}
+  ks param set jupyter platform ${KUBEFLOW_PLATFORM}
+}
+
+ksApply() {
+  pushd ${KUBEFLOW_KS_DIR}
+
+  if [ "${PLATFORM}" == "minikube" ]; then
```

Comment:
Should we always just create the Namespace? 
createNamespace should be a null op if the namespace doesn't exist.

Reply:
Agreed. I think the `if` is not needed here.